tasks/rh-sign-image-cosign/rh-sign-image-cosign.yaml

```
@@ -77,15 +77,22 @@ spec:
         run_cosign () { # Expected arguments are [digest_reference, tag_reference]
             # Upload transparency log when rekor url is specified
             if [ -v REKOR_URL ]; then
-                COSIGN_COMMON_ARGS="-y --rekor-url=$REKOR_URL --key $SIGN_KEY"
+                COSIGN_COMMON_ARGS=(
+                  -y
+                  "--rekor-url=$REKOR_URL"
+                  "--key $SIGN_KEY"
+                )
             else
-                COSIGN_COMMON_ARGS="--tlog-upload=false --key $SIGN_KEY"
+                COSIGN_COMMON_ARGS=(
+                  "--tlog-upload=false"
+                  "--key $SIGN_KEY"
+                )
```

This is wrong. This would result in passing `--key $SIGN_KEY` as one argument. BTW, you don't need `""` for anything that is just a literal string with no variables (but it's fine to have them).

```suggestion
                COSIGN_COMMON_ARGS=(
                  --tlog-upload=false
                  --key 
                  "$SIGN_KEY"
                )
```

Same applies in another place above.